src/explore-education-statistics-common/src/modules/find-statistics/components/__tests__/__snapshots__/DataBlock.test.tsx.snap

```diff
@@ -1,5 +1,217 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`DataBlock datablock renders map 1`] = `
+
+<div data-testid="DataBlock ">
+  <div class="govuk-tabs">
+    <ul class="govuk-tabs__list"
+        role="tablist"
+    >
+      <li class="govuk-tabs__list-item"
+          role="presentation"
+      >
+        <a class="govuk-tabs__tab govuk-tabs__tab--selected"
+           href="#datablock_test_charts"
+           id="datablock_test_charts-tab"
+           aria-controls="datablock_test_charts"
+           aria-selected="true"
+           role="tab"
+        >
+          Charts
+        </a>
+      </li>
+    </ul>
+    <section class="govuk-tabs__panel dfe-content-overflow panel"
+             id="datablock_test_charts"
+             data-testid="Charts"
+             aria-labelledby="datablock_test_charts-tab"
+             role="tabpanel"
+             tabindex="-1"
+    >
+      <h3>
+        Charts
+      </h3>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third"
+             aria-live="assertive"
+        >
+          <form>
+            <div class="govuk-form-group govuk-!-margin-bottom-6">
+              <label class="govuk-label"
+                     for="selectedIndicator"
+              >
+                Select data to view
+              </label>
+              <select class="govuk-select select"
+                      id="selectedIndicator"
+                      name="selectedIndicator"
+              >
+                <option value="28">
+                  Authorised absence rate
+                </option>
+                <option value="26">
+                  Overall absence rate
+                </option>
+                <option value="23">
+                  Unauthorised absence rate
+                </option>
+              </select>
+            </div>
+            <div class="govuk-form-group govuk-!-margin-bottom-6">
+              <label class="govuk-label"
+                     for="selectedLocation"
+              >
+                Select a location
+              </label>
+              <select class="govuk-select select"
+                      id="selectedLocation"
+                      name="selectedLocation"
+              >
+                <option value="E92000001">
+                  England
+                </option>
+                <option value>
+                  select...
+                </option>
+              </select>
+            </div>
+          </form>
+          <div>
+            <h3 class="govuk-heading-s">
+              Key to Authorised absence rate
+            </h3>
+            <dl class="govuk-list">
+              <dd class="legend">
+                <span class="rate0">
+                  &nbsp;
+                </span>
+                5.0%&nbsp; to 5.0%
```

Comment:
Are the rates supposed all supposed to be the same? If that's the case, we should probably just show one rate of 5.0%